src/scenes/Lobby/PlayerList/index.tsx

```diff
@@ -15,7 +15,7 @@ export default function PlayerList({ players }: PlayerListProps) {
   return (
     <div className="PlayerListSlider">
       {players.map((player) => (
-        <div key={player.id}>
+        <div key={player.avatarSeed}>
```

Comment:
algum motivo pra essa mudança? Corremos um risco de dois players com o mesmo avatar bugar aqui.

Reply:
Se ele reclamou com a key não sendo string, usa um template string e resolve.
```
<div key={`${player.id}`}>
```

Reply:
ele não reclamou, eu mudei de maneira paliativa. Não queria mexer no servidor ainda para resolver e dois players com o mesmo avatar é uma chance muito, mas muito baixa de acontecer. Quando for mexer no servidor vou resolver de vez usando o id mesmo